### ISSUE TITLE
Question about register

### ISSUE THREAD

**Issue by algomu** [TheNAF/naflm#292]
When I try to register a new user without login, Leagues are not displayed:
![image](https://user-images.githubusercontent.com/54956339/90146502-6730ea00-dd81-11ea-888f-879e9c918c27.jpeg)

When I try to modify a League with League Preferences Administration:
![image](https://user-images.githubusercontent.com/54956339/90146665-9b0c0f80-dd81-11ea-965a-6cf705546f4e.jpeg)

And finally, I try to save changes:
Oh Noes! The league administration module has failed the ptid!=stid check. Please alert your administrator!

All of this may be cause because tournament is not created? I want to create a RR Tournament with schedule matches but, to do that we need that teams were created, if coaches, can't register, can't create teams.

I miss something? Thanks for your help...

P.D.: Release in use, the last.

**Comment by algomu** [comment]
Although League Preferences Administration problem was solved when tournament was created. Register don't work yet. As the image I attached, no league is show to choose.

**Comment by byrnesvictim** [comment]
> I created a League and one division, Tournament will be created when coaches register. Can all be caused due that tournament wasn't created?

Yes I get the same error. The League Preferences Administration page cannot accept changes if the Primary Tournament field is blank. (This is not intentional and we will need to look into why). Coaches are not required to be added to a tournament when it is created, only when selecting them to schedule matches.

**Workaround; Create any tournament, this will allow in League Preferences Administration page it to be selected and OK field to work as planned.**

**Comment by byrnesvictim** [comment]
> Although League Preferences Administration problem was solved when tournament was created. Register don't work yet. As the image I attached, no league is show to choose.

I'm assuming during all this you may have deleted or created leagues trying to get this sorted out? If that is the case you will need to run the Admin - > OBBLM Core Panel tasks (4 in total). Alternatively check the troubleshoot.php and check the leagueID is the default one now? settings.php -> $settings['default_visitor_league'] = 1;  (change 1 to whatever is default)

**Comment by algomu** [comment]
It was a new install, although may be I created or deleted so but I can't remember so, I run all OBBLM Core Panel Tasks but doesn't work. With troubleshoot.php we can see that we have to leagues, and are numbered as 1 and 2.

**Comment by byrnesvictim** [comment]
Can you try change default league in settings.php I believe to 2. This is a
fresh install right? Can you try delete the Database and reinstall from
scratch?
